vocab/credentials/v2/vocabulary.yml

```diff
@@ -23,6 +23,16 @@ class:
     deprecated: true
     comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
 
+  - id: JsonSchema2023
+    label: JSON schema validator 2023
+    upper_value: cred:CredentialSchema
+    comment: A type of validator that can be used to syntactically validate JSON documents using the JSON Schema language.
+
+  - id: CredentialSchema2023
+    label: Credential schema validator 2023
+    upper_value: cred:CredentialSchema
```

Comment:
```suggestion
```

This doesn't seem to be defined anywhere. It really doesn't need to be... any reason you're wanting to define a base class (other than for completeness)?